repos/spack_repo/builtin/packages/tempestextremes/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+
+from spack.package import *
+
+
+class Tempestextremes(CMakePackage):
+    """TempestExtremes is a growing collection of detection and
+    characterization algorithms for large climate datasets, leveraging C++ for
+    rapid throughput and a command line interface that maximizes flexibility
+    of each kernel. The tracking kernels in this package have been already
+    used for tracking and characterizing tropical cyclones (TCs), extratropical
+    cyclones (ETCs), monsoonal depressions, atmospheric blocks, atmospheric
+    rivers, and mesoscale convective systems (MCSs). By considering multiple
+    extremes within the same framework, we can study the joint characteristics
+    of extremes while minimizing the total data burden.
+    """
+
+    homepage = "https://github.com/ClimateGlobalChange/tempestextremes"
+    url = "https://github.com/ClimateGlobalChange/tempestextremes/archive/refs/tags/v2.3.tar.gz"
+
+    maintainers("andrewdnolan", "paullric", "xylar")
```

Comment:
@paullrich, let us know if you'd rather not be a maintainer.  We're including you on the assumption you might want to keep an eye on things.